eva_cttv_pipeline/trait_mapping/oxo.py

```diff
@@ -20,13 +20,15 @@ class OntologyUri:
         "medgen": "http://identifiers.org/medgen/{}",
         "human phenotype ontology": "http://purl.obolibrary.org/obo/HP_{}",
         "hp": "http://purl.obolibrary.org/obo/HP_{}",
-        "doid": "http://purl.obolibrary.org/obo/DOID_{}"
+        "doid": "http://purl.obolibrary.org/obo/DOID_{}",
+        "mondo": "http://purl.obolibrary.org/obo/MONDO_{}",
     }
 
     def __init__(self, id_, db):
         self.id_ = id_
         self.db = db
         if self.db.lower() == "human phenotype ontology":
+            logging.warning('Found <human phenotype ontology>, this should not really happen')
```

Comment:
This warning is added because I haven't seen OxO return such a database (`human phenotype ontology`, like that, with spaces); is this part of the code obsolete and should be removed?

Reply:
It has been present since the very first version of the code, so maybe it is obsolete. Have you checked it doesn't show up even once after running the pipeline?

Reply:
Yes, I can confirm that this warning does not appear in the logs at all, hence this branch is never reached. Removed it from the code entirely.